data-extraction/data-import/src/main/java/com/wipro/ats/bdre/dataimport/HDFSImport.java

```diff
@@ -211,29 +207,19 @@ public int run(String[] param) throws Exception {
                     OozieUtil oozieUtil = new OozieUtil();
                     oozieUtil.persistBeanData(registerFileInfo, false);
 
-                    if (null != commonProperties.getProperty("query") && "" != commonProperties.getProperty("query")) {
+                    if (null != commonProperties.getProperty(QUERY_STRING) && "" != commonProperties.getProperty(QUERY_STRING)) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)